Local Pods/SlackTextViewController/Source/SLKTextViewController.m

```diff
@@ -2260,16 +2260,19 @@ - (void)slk_setupViewConstraints
     
     [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|[scrollView(0@750)][typingIndicatorView(0)]-0@999-[textInputbar(0)]|" options:0 metrics:nil views:views]];
     [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-(>=0)-[autoCompletionView(0@750)][typingIndicatorView]" options:0 metrics:nil views:views]];
-//    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[scrollView]|" options:0 metrics:nil views:views]];
     [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[autoCompletionView]|" options:0 metrics:nil views:views]];
     [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[typingIndicatorView]|" options:0 metrics:nil views:views]];
     [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[textInputbar]|" options:0 metrics:nil views:views]];
 
-    NSLayoutConstraint *leadingSafe = [self.scrollViewProxy.leadingAnchor constraintEqualToSystemSpacingAfterAnchor:self.view.safeAreaLayoutGuide.leadingAnchor multiplier:1.0];
-    leadingSafe.constant = 0;
-    NSLayoutConstraint *trailingSafe = [self.scrollViewProxy.trailingAnchor constraintEqualToSystemSpacingAfterAnchor:self.view.safeAreaLayoutGuide.trailingAnchor multiplier:1.0];
-    trailingSafe.constant = 0;
-    [NSLayoutConstraint activateConstraints:@[leadingSafe, trailingSafe]];
+    if (@available(iOS 11, *)) {
+        NSLayoutConstraint *leadingSafe = [self.scrollViewProxy.leadingAnchor constraintEqualToSystemSpacingAfterAnchor:self.view.safeAreaLayoutGuide.leadingAnchor multiplier:1.0];
+        leadingSafe.constant = 0;
+        NSLayoutConstraint *trailingSafe = [self.scrollViewProxy.trailingAnchor constraintEqualToSystemSpacingAfterAnchor:self.view.safeAreaLayoutGuide.trailingAnchor multiplier:1.0];
+        trailingSafe.constant = 0;
+        [NSLayoutConstraint activateConstraints:@[leadingSafe, trailingSafe]];
+    } else {
+        [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[scrollView]|" options:0 metrics:nil views:views]];
```

Comment:
Not sure if you removed this one for another reason? Should we drop it completely?